WordPressCom-Stats-iOS/WPStatsService.m

```diff
@@ -78,7 +78,7 @@ - (void)retrieveAllStatsForDate:(NSDate *)date
     NSMutableDictionary *cacheDictionary = [self.ephemory objectForKey:@[@"BatchStats", @(unit), endDate]];
     DDLogVerbose(@"Cache count: %@", @(cacheDictionary.count));
     
-    if (cacheDictionary && cacheDictionary.count == 13) {
+    if (cacheDictionary && cacheDictionary.count == 15) {
```

Comment:
(This one could really use a constant + a comment why. No need to check if cacheDictionary is not nil, since nil.count == 0)